### ISSUE TITLE
finditer() bug when calling project_data_from_master()

### ISSUE THREAD

**Issue by hammerheadlemon** [hammerheadlemon/bcompiler#2]
Trace:

```python
Traceback (most recent call last):
 File "<input>", line 1, in <module>
 File "C:\Program Files\JetBrains\PyCharm Community Edition 2018.3.5\helpers\pydev\_pydev_bundle\pydev_umd.py", line 197, in runfile
 pydev_imports.execfile(filename, global_vars, local_vars) # execute the script
 File "C:\Program Files\JetBrains\PyCharm Community Edition 2018.3.5\helpers\pydev\_pydev_imps\_pydev_execfile.py", line 18, in execfile
 exec(compile(contents+"\n", file, 'exec'), glob, loc)
 File "C:/Users/Standalone/PycharmProjects/Portfolio_Finance/aggregate_financial_profile.py", line 261, in <module>
 q_one = project_data_from_master("C:\\Users\\Standalone\\Will\\masters folder\\core data\\Hs2_NPR_Q1_1918_draft.xlsx")
 File "C:\Users\Standalone\PycharmProjects\Portfolio_Finance\venv\lib\site-packages\bcompiler\utils.py", line 219, in project_data_from_master
 c = Cleanser(cell.value)
 File "C:\Users\Standalone\PycharmProjects\Portfolio_Finance\venv\lib\site-packages\bcompiler\process\cleansers.py", line 134, in __init__
 self._analyse()
 File "C:\Users\Standalone\PycharmProjects\Portfolio_Finance\venv\lib\site-packages\bcompiler\process\cleansers.py", line 265, in _analyse
 matches = re.finditer(self._checks[i]['rule'], self.string)
 File "C:\Users\Standalone\AppData\Local\Programs\Python\Python36-32\lib\re.py", line 229, in finditer
 return _compile(pattern, flags).finditer(string)
TypeError: expected string or bytes-like object
```

**Comment by hammerheadlemon** [comment]
This is caused by openpyxl column overflow beyond the end of the data. Will update `bcompiler` to handle this exception.

**Comment by hammerheadlemon** [comment]
Fixed in 47f955a1275004437fac2f4f0119788e5e48d6a4.  In `utils.py`:

```python
    for cell in ws["A"]:
        # we don't want to clean None...
        if cell.value is None:
            continue
```
